.github/workflows/build_package.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Deploy
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install tree CLI tool
+      run: sudo apt-get install -y tree
+
+    - name: Display directory tree before building
+      run: tree
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
+    - name: Set up Conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: env.yml
+        activate-environment: mle-dev
+
+    - name: Install Poetry
+      run: |
+        curl -sSL https://install.python-poetry.org | python3 -
+        export PATH="$HOME/.poetry/bin:$PATH"
+
+    - name: Install dependencies
+      run: poetry install
+
+    - name: Build the package
+      run: poetry build
+
+    - name: Display directory tree after building
+      run: tree
+
+    - name: Check contents of dist directory
+      run: ls -la
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: build-artifacts
+        path: dist/*
+
+  deploy_package:
+    runs-on: ubuntu-latest
+
+    needs: build_package
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.11'
+
+    - name: Set up Conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: env.yml
+        activate-environment: mle-dev
+
+    - name: Download build artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: build-artifacts
+        path: dist  # Specify the path to download artifacts to
+
+    - name: List downloaded artifacts
+      run: ls -la dist
+
+    - name: Install the package
+      run: |
+        cd dist
+        pip install *.whl
+
+    - name: Run housing_price.py
+      run: python src/house_package_saachi/housing_price.py
```

Comment:
Run the code from the installed package not from `src/`.
So you will have to do something like this..
`python -m house_package_saachi.house_price`

Reply:
Run the package and change the workflow